File: pkg/codegen/testing/test/testdata/transpiled_examples/cue-static-web-app-pp/python/cue-static-web-app.py
import pulumi
import pulumi_azure_native as azure_native

rawkode_group = azure_native.resources.ResourceGroup("rawkode-group", location="WestUs")
rawkode_storage = azure_native.storage.StorageAccount("rawkode-storage",
    resource_group_name=rawkode_group.name,
    kind="StorageV2",
    sku=azure_native.storage.SkuArgs(
        name="Standard_LRS",
    ))
rawkode_website = azure_native.storage.StorageAccountStaticWebsite("rawkode-website",
    resource_group_name=rawkode_group.name,
    account_name=rawkode_storage.name,
    index_document="index.html",
    error404_document="404.html")
rawkode_index_html = azure_native.storage.Blob("rawkode-index.html",
    resource_group_name=rawkode_group.name,
    account_name=rawkode_storage.name,
    container_name=rawkode_website.container_name,
    content_type="text/html",
    type=azure_native.storage.BlobType.BLOCK,
    source=pulumi.FileAsset("./website/index.html"))
stack72_group = azure_native.resources.ResourceGroup("stack72-group", location="WestUs")
stack72_storage = azure_native.storage.StorageAccount("stack72-storage",
    resource_group_name=stack72_group.name,
    kind="StorageV2",
    sku=azure_native.storage.SkuArgs(
        name="Standard_LRS",
    ))
stack72_website = azure_native.storage.StorageAccountStaticWebsite("stack72-website",
    resource_group_name=stack72_group.name,
    account_name=stack72_storage.name,
    index_document="index.html",
    error404_document="404.html")
stack72_index_html = azure_native.storage.Blob("stack72-index.html",
    resource_group_name=stack72_group.name,
    account_name=stack72_storage.name,
    container_name=stack72_website.container_name,
    content_type="text/html",
    type=azure_native.storage.BlobType.BLOCK,
    source=pulumi.FileAsset("./website/index.html"))

File: pkg/codegen/testing/test/testdata/transpiled_examples/cue-static-web-app-pp/go/cue-static-web-app.go
package main

import (
	resources "github.com/pulumi/pulumi-azure-native/sdk/go/azure/resources"
	storage "github.com/pulumi/pulumi-azure-native/sdk/go/azure/storage"
	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
)

func main() {
	pulumi.Run(func(ctx *pulumi.Context) error {
		rawkodeGroup, err := resources.NewResourceGroup(ctx, "rawkode-group", &resources.ResourceGroupArgs{
			Location: pulumi.String("WestUs"),
		})
		if err != nil {
			return err
		}
		rawkodeStorage, err := storage.NewStorageAccount(ctx, "rawkode-storage", &storage.StorageAccountArgs{
			ResourceGroupName: rawkodeGroup.Name,
			Kind:              pulumi.String("StorageV2"),
			Sku: &storage.SkuArgs{
				Name: pulumi.String("Standard_LRS"),
			},
		})
		if err != nil {
			return err
		}
		rawkodeWebsite, err := storage.NewStorageAccountStaticWebsite(ctx, "rawkode-website", &storage.StorageAccountStaticWebsiteArgs{
			ResourceGroupName: rawkodeGroup.Name,
			AccountName:       rawkodeStorage.Name,
			IndexDocument:     pulumi.String("index.html"),
			Error404Document:  pulumi.String("404.html"),
		})
		if err != nil {
			return err
		}
		_, err = storage.NewBlob(ctx, "rawkode-index.html", &storage.BlobArgs{
			ResourceGroupName: rawkodeGroup.Name,
			AccountName:       rawkodeStorage.Name,
			ContainerName:     rawkodeWebsite.ContainerName,
			ContentType:       pulumi.String("text/html"),
			Type:              storage.BlobTypeBlock,
			Source:            pulumi.NewFileAsset("./website/index.html"),
		})
		if err != nil {
			return err
		}
		stack72Group, err := resources.NewResourceGroup(ctx, "stack72-group", &resources.ResourceGroupArgs{
			Location: pulumi.String("WestUs"),
		})
		if err != nil {
			return err
		}
		stack72Storage, err := storage.NewStorageAccount(ctx, "stack72-storage", &storage.StorageAccountArgs{
			ResourceGroupName: stack72Group.Name,
			Kind:              pulumi.String("StorageV2"),
			Sku: &storage.SkuArgs{
				Name: pulumi.String("Standard_LRS"),
			},
		})
		if err != nil {
			return err
		}
		stack72Website, err := storage.NewStorageAccountStaticWebsite(ctx, "stack72-website", &storage.StorageAccountStaticWebsiteArgs{
			ResourceGroupName: stack72Group.Name,
			AccountName:       stack72Storage.Name,
			IndexDocument:     pulumi.String("index.html"),
			Error404Document:  pulumi.String("404.html"),
		})
		if err != nil {
			return err
		}
		_, err = storage.NewBlob(ctx, "stack72-index.html", &storage.BlobArgs{
			ResourceGroupName: stack72Group.Name,
			AccountName:       stack72Storage.Name,
			ContainerName:     stack72Website.ContainerName,
			ContentType:       pulumi.String("text/html"),
			Type:              storage.BlobTypeBlock,
			Source:            pulumi.NewFileAsset("./website/index.html"),
		})
		if err != nil {
			return err
		}
		return nil
	})
}
